app/default/data.py

```diff
@@ -168,17 +175,21 @@ def get_applications_for_account(account_id, as_dict=False):
     )
 
 
-def get_fund_data(fund_id, language=None, as_dict=False):
+@lru_cache(maxsize=5)
+def get_fund_data(fund_id, language=None, as_dict=False, ttl_hash=None):
+    del ttl_hash  # Only needed for lru_cache
     language = {"language": language or get_lang()}
     fund_request_url = Config.GET_FUND_DATA_ENDPOINT.format(fund_id=fund_id)
     fund_response = get_data(fund_request_url, language)
     if as_dict:
-        return Fund.from_dict(fund_response)
-    else:
         return fund_response
+    else:
+        return Fund.from_dict(fund_response)
 
 
-def get_fund_data_by_short_name(fund_short_name, as_dict=False):
+@lru_cache(maxsize=5)
+def get_fund_data_by_short_name(fund_short_name, as_dict=False, ttl_hash=None):
```

Comment:
For consistency, these should probably be separated onto their own lines, as [seen below](https://github.com/communitiesuk/funding-service-design-frontend/pull/366/files#diff-25ffeea551d125947892e07d498380fe030e5f7c005ba15e165622de5d567f0eR242)